src/main/java/com/sprints/model/Sprint.java

```diff
@@ -1,5 +1,103 @@
 package com.sprints.model;
 
+import java.util.Date;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection="sprints")
 public class Sprint {
+	@Id
+	String id;
+	public String getId() {
+		return id;
+	}
+
+
+
+	String technology;
+	boolean active;
+	boolean is_backlog;
+	Date start_date;
+	Date end_date;
```

Comment:
**Always** use `LocalDate `or `LocalDateTime` instead of Date.

Reply:
Done